src/app/api/portfolio/[id]/route.ts

```diff
@@ -47,6 +44,15 @@ export async function GET(request: NextRequest) {
         });
     }
 
+    const userId = userData.user.id as string;
+    const role = userData.user.role as string;
+
+    if (role === "customer" && data.owner_id !== userId) {
```

Comment:
Müsste das nicht "authenticated" statt "customer" sein?
```suggestion
    if (role === "authenticated" && data.owner_id !== userId) {
```

Reply:
Jo true, hab angepasst